internal/ethapi/api.go

```diff
@@ -1669,7 +1669,7 @@ func (s *PublicTransactionPoolAPI) GetTransactionReceipt(ctx context.Context, ha
 		"logs":              receipt.Logs,
 		"logsBloom":         receipt.Bloom,
 		"type":              hexutil.Uint(tx.Type()),
-		"l1Fee":             hexutil.Uint64(receipt.L1Fee.Uint64()),
+		"l1Fee":             (*hexutil.Big)(receipt.L1Fee),
```

Comment:
`blockNumber`, `transactionIndex`, etc all use `hexutil.Uint64`, so let's keep `l1Fee` consistent with them.

Reply:
But in [marshal and unmarshal](https://github.com/scroll-tech/go-ethereum/blob/910d4f1a70fc29d4433b9c793191e29203489d9d/core/types/gen_receipt_json.go#L68) interface we use `*hexutil.Big` type, this is inconsistent.

Reply:
`BlockNumber` in `UnmarshalJSON` also uses `*hexutil.Big`, does that also lead to issues?

Reply:
It's not a bug, this PR is just a suggested modification.

Reply:
What about changing "blockNumber" too?

Reply:
`BlockNumber`'s type is uint64 in `GetTransactionReceipt`. So it's no need to change the type again.

Reply:
I'd prefer to keep it consistent with how geth handles other fields.

Reply:
You're right, it's better to use `*hexutil.Big` to avoid overflow.